run_uh.py

```diff
@@ -294,53 +295,55 @@ def import_fife(paths):
 		return False
 	return True
 
-def find_fife():
-	# Use the path the user provided.
+def get_fife_paths():
+	"""Returns all possible paths to search for the fife module. New paths to be added as needed."""
+	# Use the path the user provided:
 	from horizons.util.cmdlineoptions import get_option_parser
 	options = get_option_parser().parse_args()[0]
+	# All paths collected in a list:
```

Comment:
I don't think this comment improves the code, `paths = []` is already pretty clear. I'd remove it.